contrib/build-linux/appimage/_build.sh

```diff
@@ -145,6 +145,9 @@ info "Finalizing AppDir"
     mv usr/include.tmp usr/include
 ) || fail "Could not finalize AppDir"
 
+# We copy libusb here because it is on the AppImage excludelist and it can cause problems if we use system libusb
+info "Copying libusb"
+cp /usr/lib/x86_64-linux-gnu/libusb-1.0.so "$APPDIR/usr/lib/libusb-1.0.so" || fail "Could not copy libusb"
```

Comment:
Might want to do `cp -f` here out of paranoia in case a future AppImage also includes it.  Dunno.. up to you. Likely the other `cp` calls in this script don't do `-f` .. I forget. :)

Reply:
Maybe even `-p`? I forget if `+x` is preserve on copy by default (it is, right?).

Reply:
Done